gtfs2netexfr/src/main.rs

```diff
@@ -26,35 +26,44 @@ use transit_model::Result;
 #[derive(Debug, StructOpt)]
 #[structopt(name = "gtfs2netexfr", about = "Convert a GTFS to NeTEx France.")]
 struct Opt {
-    /// input directory.
+    /// Input directory.
     #[structopt(short, long, parse(from_os_str), default_value = ".")]
     input: PathBuf,
 
-    /// output directory
+    /// Output directory
     #[structopt(short, long, parse(from_os_str))]
     output: PathBuf,
 
-    /// config file
-    #[structopt(short, long, parse(from_os_str))]
+    /// Config file
+    #[structopt(
+        short,
+        long,
+        parse(from_os_str),
+        long_help = r"JSON file containing additionnal configuration.
+
+See https://github.com/CanalTP/transit_model/blob/master/documentation/common_ntfs_rules.md#configuration-of-each-converter
+for more information.
+"
+    )]
     config: Option<PathBuf>,
 
-    /// OnDemandTransport GTFS source
+    /// Indicates if the input GTFS contains On-demand Transport (ODT) information
```

Comment:
minor:
```suggestion
    /// Indicates if the input GTFS contains On-Demand Transport (ODT) information
```

same below.